components/eam/bld/namelist_files/namelist_definition.xml

```diff
@@ -4494,6 +4494,12 @@ Return fluxes per band in addition to the total fluxes.
 Default: FALSE
 </entry>
 
+<entry id="rad_asym_splt" type="logical"  category="radiation"
+group="radiation_nl" valid_values="" > 
+Split radiation asymmetrically between VIS/NIR, coef for VIS
```

Comment:
"Split surface insolation asymmetrically between VIS/NIR, coef for VIS" would be clearer